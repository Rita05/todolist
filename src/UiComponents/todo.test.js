
import { render } from "@testing-library/react";
import renderer from 'react-test-renderer';

import Todo from './todo';

describe("Making sure it works", () => {
  it("test render", () => {
    const props = {
      tasks: [
        {
          id: 1, 
          taskName: "testTask"
        }
      ],
      task: "Buy Products",
      onTaskÐ¡hanged: () => console.log("app adding tasks"), 
      onRemoveTask: () => {}, 
      onAddTask: () => {}
    }
    render(<Todo {...props}/>);
  })
  it("test without props", () => {
    render(<Todo />);
  })
  it('render correctly input task from', () => {
    const TodoListComponent = renderer.create(<Todo/>).toJSON();
    expect(TodoListComponent).toMatchSnapshot();
  });

})
