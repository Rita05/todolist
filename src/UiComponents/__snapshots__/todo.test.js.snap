// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Making sure it works render correctly input task from 1`] = `
<div
  className="flex flex-col w-full max-w-500px mt-100px p-30px rounded-10px bg-zinc-100"
>
  <h1
    className="uppercase text-center font-serif italic text-green-400 font-bold"
  >
    my todolist
  </h1>
  <div
    className="flex justify-center items-center space-x-20px pt-20px"
  >
    <input
      className="w-20rem h-2.5rem p-5px border-1 rounded-md"
      onChange={[Function]}
      placeholder="Enter Task"
      type="text"
    />
    <button
      className="bg-teal-400 w-8rem h-2.5rem rounded-md"
      onClick={[Function]}
    >
      Add task
    </button>
  </div>
  <div
    className="flex flex-col justify-center items-center space-y-5 m-20px"
  />
</div>
`;
